ofl/grandstander/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,7 @@
+<p>
+<em>Grandstander</em> is a display variable font with a weight axis. 9 weights, upright and italic. It supports a wide range of languages in the latin script scope.
+</p>
+<p>
+The Epilogue project is led by Tyler Finck — type designer running <a href="https://www.etc.supply">Etcetera Type Co</a> in Ithaca, New-York, USA.
```

Comment:
Is Grandstander part of the Epilogue project?

Reply:
Ah haha, I forgot this one…